ClientTest/src/com/rain/utils/android/robocop/clienttest/Main.java

```diff
@@ -11,6 +11,6 @@
 public class Main {
     public static void main(String[] args) {
         ContentProviderWriter writer = new ContentProviderWriter();
-        writer.createContentProvider("ClientTest/resources/example_schema.json", "ClientTest/src-gen/");
+        writer.createContentProvider("ClientTest/resources/classes_only_schema.json", "ClientTest/src-gen/");
```

Comment:
should this be split into two separate tests instead of replacing an existing one? (do we run these tests?)